gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreTest.java

```diff
@@ -68,7 +67,11 @@
 @RunWith(JUnit4.class)
 public class DatastoreTest {
 
-  private static final String PROJECT_ID = LocalGcdHelper.DEFAULT_PROJECT_ID;
+  private static LocalDatastoreHelper helper = LocalDatastoreHelper.create(1.0);
+  private static final DatastoreOptions options =
+      helper.options().toBuilder().retryParams(RetryParams.noRetries()).build();
```

Comment:
Any reason for not using `RetryParams.noRetries()` by default for the local service helper?